website/docs/language/settings/backends/gcs.mdx

```diff
@@ -86,7 +108,7 @@ The following configuration options are supported:
   (Optional) Local path to Google Cloud Platform account credentials in JSON
   format.  If unset, [Google Application Default
   Credentials](https://developers.google.com/identity/protocols/application-default-credentials)
-  are used.  The provided credentials must have Storage Object Admin role on the bucket.
+  are used.  The provided credentials must have the Storage Object Admin role on the bucket.
```

Comment:
I can't include a direct suggest change across a deletion. So I suggest rewording the sentence to "If unset, the path uses [Google Application Default Credentials]..." to avoid a passive voice style issue.

Reply:
Will do, thanks!